acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/web3j/privacy/PrivacyGroupAcceptanceTest.java

```diff
@@ -14,121 +14,101 @@
 
```

Comment:
it may be worth breaking out the participants as a variable, and confirming the content of the privacyGroup.

Testing that "Default Name" comes in here is really testing Orion functionality - not Pantheon.
Secondly - if an arbitrary name is going to be applied to the group, I'd argue that a blank string may be more appropriate.

Reply:
confirming the content of privacy group complete

the second comment here is specific to Orion - no action point to be taken here